_includes/nav.html

```diff
@@ -1,20 +1,18 @@
-<nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
-    <div class="container">
```

Comment:
I removed the container because the `navbar` should not contains it, otherwise it results in a not-fully-working navbar.